tfx/components/util/udf_utils.py

```diff
@@ -295,24 +295,29 @@ class TempPipInstallContext:
 
   def __init__(self, pip_dependencies: List[str]):
     if not isinstance(pip_dependencies, list):
-      raise ValueError('Expected list of dependencies, got %r instead.' %
-                       (pip_dependencies,))
+      raise ValueError('Expected list of dependencies, '
+                       f'got {pip_dependencies!r} instead.')
     self.pip_dependencies = pip_dependencies
     self.temp_directory = None
+    self.orig_syspath = None
+    self.orig_pypath = None
 
   def __enter__(self) -> 'TempPipInstallContext':
     if self.pip_dependencies:
       self.temp_directory = tempfile.mkdtemp()
       for dependency in self.pip_dependencies:
         install_to_temp_directory(dependency, temp_dir=self.temp_directory)
+      self.orig_syspath = sys.path
+      self.orig_pypath = os.environ['PYTHONPATH']
       sys.path = sys.path + [self.temp_directory]
-      os.environ['PYTHONPATH'] = ':'.join(sys.path)
+      os.environ['PYTHONPATH'] = self.orig_pypath + \
```

Comment:
I'm sorry but backslash is not allowed in google style guide. Could you replace it with parentheses?

https://google.github.io/styleguide/pyguide.html#32-line-length